src/ccontrol/UserQuerySelect.cc

```diff
@@ -607,6 +612,18 @@ void UserQuerySelect::_qMetaUpdateStatus(qmeta::QInfo::QStatus qStatus) {
     }
 }
 
+void UserQuerySelect::_qMetaUpdateMessages() {
+    // message table
+
+    auto msgStore = getMessageStore();
+    try {
+        _queryMetadata->addQueryMessages(_qMetaQueryId, msgStore);
+    } catch (qmeta::SqlError const& ex) {
+        LOGS(_log, LOG_LVL_ERROR, "UserQuerySelect::_qMetaUpdateMessages failed " << ex.what());
```

Comment:
Should this be treated as `WARN` since the code is not re-throwing the exception or taking any other measures to address the problem?
In theory, if we can't update any table in the database this should be a reason to raise an alarm.